edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/ztp-image-based-upgrade-prep-resources.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cnf-image-based-upgrade-prep-resources"]
+= Creating ConfigMap objects for the image-based upgrade with {lcao} using {ztp}
+include::_attributes/common-attributes.adoc[]
+:context: ztp-gitops
+
+toc::[]
+
+The {lcao} needs all your OADP resources, extra manifests, and custom catalog sources wrapped in a `ConfigMap` object to process them for the image-based upgrade.
+
+:FeatureName: The Lifecycle Agent
+
+include::modules/ztp-image-based-upgrade-prep-oadp.adoc[leveloffset=+1]
+
+
+include::modules/ztp-image-based-upgrade-prep-label-extramanifests.adoc[leveloffset=+1]
```

Comment:
The alignment doesn't look right in the doc preview

Reply:
Fixed